tailwind.config.js

```diff
@@ -1,19 +1,45 @@
+function withOpacityValue(variable) {
+  return ({ opacityValue }) => {
+    if (opacityValue === undefined) {
+      console.log(`rgb(var(${variable}))`)
+      return `rgb(var(${variable}))`
+    }
+    return `rgb(var(${variable}) / ${opacityValue})`
+  }
+}
+
 module.exports = {
-  mode: "jit",
-  content: ["./src/**/*.{js,ts,jsx,tsx}"],
+  content: ["./src/**/*.{js,ts,jsx,tsx,css}"],
   theme: {
     extend: {
       colors: {
-        orangeweboxfordblue: {
-          background: "#FFFFFF",
-          border: "#000000",
-          primary: "#14213D",
-          secondary: "#FFFFFF",
-          tertiary: "#FCA311",
-          quaternary: "#E5E5E5",
-        },
+        skin: {
```

Comment:
Just curious--is "skin" the tailwind best practice for naming themes?

Reply:
This was following a tailwind youtube channel, which used this naming convention. I think with the implementation here, it's useful as a reminder that it's an extension, rather than a replacement of the core Tailwind utilities.